protos/bottle/account/events/v1/organization_left.proto

```diff
@@ -1,9 +1,9 @@
 syntax = "proto3";
 
-package bottle.notification.user.v1;
+package bottle.account.events.v1;
 
 import "bottle/account/v1/user.proto";
 
-message PasswordChanged {
+message OrganizationLeft {
```

Comment:
We should also include the organization in this message

Reply:
I've been using it that way in the service code, need to move that here 👍